helm/templates/deployment.yaml

```diff
@@ -91,15 +102,15 @@ spec:
                   name: {{ $s3SecretName }}
                   key: secretAccessKey
             {{- end }}
-            {{- if eq (upper .Values.rasterCommon.storage.mapproxyConfigProvider) "FS" }}
+            {{- if eq (upper $storage.mapproxyConfigProvider) "FS" }}
             - name: MAPPROXY_YAML_FILE_PATH  
               value: /configSource/mapproxy.yaml
             {{- end }} 
             - name: INTERNAL_MOUNT_DIR
```

Comment:
isn't INTERNAL_MOUNT_DIR and ROOT_TILES_PATH uses for mapproxy config location? if its for FS why isn't it inside the If? - please check the need for them

Reply:
bug from the past, its not on the scope of the PR but fixed